biothings/hub/datatransform/datatransform.py

```diff
@@ -39,6 +39,8 @@ def add(self, left, right):
         """add a (original_id, current_id) pair to the list"""
         if not left or not right:
             return  # identifiers cannot be None
+        if self.lookup(left, right):
```

Comment:
did moving this test there fix the issue ? why ?

Reply:
Yes, I think it did.  The lookup method takes as arguments the values and checks if they are in the structure.  Before moving this test, the lookup method was receiving tuples or lists that are part of the structure itself.

Reply:
not sure to understand but I'll blindly merge that PR...